packages/oas/test/operation/index.test.ts

```diff
@@ -1072,7 +1072,7 @@ describe('#getOperationId()', () => {
       });
 
       const operation = spec.operation('/ac_eq_hazard/18.0', 'post');
-      expect(operation.getOperationId({ camelCase: true })).toBe('postAc_eq_hazard180');
+      expect(operation.getOperationId({ camelCase: true })).toBe('postAcEqHazard180');
```

Comment:
In our new Git world `postAcEqHazard180.md` will end up being the filename for this endpoint. It's still not great but is at least a bit cleaner than `postAc_eq_hazard180.md`